lib/tlGL/RenderShaders.cpp

```diff
@@ -333,12 +333,12 @@ namespace tl
                 "    tmp[0] = (value[0] - data.inLow) / data.inHigh;\n"
                 "    tmp[1] = (value[1] - data.inLow) / data.inHigh;\n"
                 "    tmp[2] = (value[2] - data.inLow) / data.inHigh;\n"
-                "    if (tmp[0] >= 0.0)\n"
-                "        tmp[0] = pow(tmp[0], data.gamma);\n"
-                "    if (tmp[1] >= 0.0)\n"
-                "        tmp[1] = pow(tmp[1], data.gamma);\n"
-                "    if (tmp[2] >= 0.0)\n"
-                "        tmp[2] = pow(tmp[2], data.gamma);\n"
+                // "    if (tmp[0] >= 0.0)\n"
```

Comment:
The exposure controls are meant to be used as an alternative to the other controls, so this code should be left in place. After this PR is finished I will rename the exposure code to something like "exrdisplay" and add a comment about it's usage.